xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
It is strange though, all I did was remove the offScreen part, which was really straight forward. So either I made a very small typo which I am obviously overlooking here or there is another place in the code where the constructor for the GuiLock is defined but that doesn't make sense to me.